truncate_client/src/utils/daily.rs

```diff
@@ -57,22 +57,16 @@ pub fn get_daily_puzzle(
     let seed = (local_seconds / (60 * 60 * 24)) as u32;
     let day = seed - DAILY_PUZZLE_DAY_ZERO as u32;
     let mut board_seed = BoardSeed::new(seed).day(day);
-    let persisted_moves = get_persistent_game(&board_seed);
```

Comment:
We were doing this too early, before `board_seed.external_reroll()` below, which meant on days where the seed was rerolled we wouldn't load the moves and the puzzle would restart. Moved til after we do the rerolls.